### ISSUE TITLE
controlNet - convert sketch circuit to vector-engineered circuit,some question

### ISSUE THREAD

**Issue by JaosonMa** [lllyasviel/ControlNet#526]
I want to achieve what is shown in the diagram below, converting a hand-drawn circuit diagram into a vector-engineered diagram. eg.
![image](https://github.com/lllyasviel/ControlNet/assets/23551774/1ff4c25b-5f3c-4bf5-85f0-10d98fcf8cb7)

all the controlText is : {"text": "Convert this sketch circuit diagram into a standard vectorized circuit diagram created with software", "image": "images/1000_1_360015968.jpg", "conditioning_image": "conditioning_images/1000_1_360015968.jpg"}
I have prepared 50,000 similar data pairs, with a sketch circuit diagram corresponding to an identical vector image. 
Then, I trained ControlNet with
```
batchsize=1(my gpu is one 3090), 
lr = 2e-6, 
precision=32,
accumulate_grad_batches=4,
sd_locked = False,
only_mid_control = False, 
```
Roughly after 2 epochs, there wasn't much change in the test simple images. The input sketch didn't seem to have any controlling effect, and there was a tendency for overfitting. Regardless of the input sketch, the output results were random combinations of circuit diagrams.
here are some egs:
text: Convert this sketch circuit diagram into a standard vectorized circuit diagram created with software
control image:
![image](https://github.com/lllyasviel/ControlNet/assets/23551774/6025e4f0-b1cd-4dd3-992b-e2428c9d1e96)
output image:
![image](https://github.com/lllyasviel/ControlNet/assets/23551774/03965144-9bd0-4281-af99-c00fcec55ed4)

can some one tell me some advice?  thanks!

**Comment by engrmusawarali** [comment]
Hello!

First of all lock your SD. Which seems to to be unlocked. Second use increased batch size i.e 64, using the gradient accumulation. Increased batch size will cost you more training time but better results.

**Comment by CQxiaocaimi** [comment]
Friend, have you solved your problem?  I am also facing the same problem as you, but I still can't find the answer.  So I want to ask you, maybe you can give me some hints and help.Thank you.I want to control the output image just like you.  I tried the method mentioned in the comments, but it didn't work.  The output images are random and chaotic, and have nothing to do with the input image.I'm going crazy.

**Comment by xzwj8666** [comment]
I have similar problems. Have you solved it?

**Comment by Kusunoki0130** [comment]
Maybe Diffusion Model is hard to recognize circuit components as it to texts or charaters，when you try to reconstruct images with them.  

个人猜测：电路元件比较像文字符号，diffusion 模型这一系列我感觉它看不懂文字符号，没有办法细致地重构图像里文字，同样也不能很好地理解电路元件的排列。
